scripts/lib/build-utils.mjs

```diff
@@ -288,6 +288,23 @@ export default {
     });
   },
 
+  /**
+   * Build the vtunnel.
+   */
+  async buildVtunnel(platform) {
+    const target = platform === 'win32' ? 'vtunnel.exe' : 'vtunnel';
+    const parentDir = path.join(this.srcDir, 'resources', platform, 'bin');
```

Comment:
`vtunnel` should go into `resources/*/internal`, not `resources/*/bin` (`bin` is for things the user might run, whereas `internal` is for things we run).